src/module.ts

```diff
@@ -62,6 +72,7 @@ const defaults: ModuleOptions & { basePath: string } = {
   isEnabled: true,
   origin: undefined,
   basePath: '/api/auth',
+  trustHost: Boolean(process.env.AUTH_TRUST_HOST ?? process.env.VERCEL),
```

Comment:
I think this will not work as expected: This is only evaluated at "build time" of the module AFAIK. So then when time for deployment rolls around, `trustHost` option is _not_ going to be configurable via `process.env.AUTH_TRUST_HOST` any longer.

Instead, we'll have to read this at runtime, AFAIU we can:
- process this at runtime 
- add it to the runtimeConfig, so that it is picked up by nuxt from the env on startup (see https://nuxt.com/docs/migration/runtime-config#runtime-config)

I think the second option is better.

This topic in general imho is a bit messy in nuxt 3 with how modules handle config + runtime config + env vars.

Reply:
Yeah, let's add the env for now and see if there's a later request for env-variable support and then work on it.